Python tool-cache open-source (full-changes) (#42)
akv-platform/toolcache-python-generation: README.md

> # python-for-virtual-environments

Reviewer: we should change this to "Python for Actions"

> This repository contains the code and scripts that we use to build Python packages used in [virtual-environments](https://github.com/actions/virtual-environments).  

Reviewer: also add "and accessible through the [setup-python](url) Action"

File [versions-manifest.json](./versions-manifest.json) contains the list of available and released versions.  

Some versions are pre-installed on [virtual-environments](https://github.com/actions/virtual-environments) images.  

> All available versions will (soon!) be available to install on-fly via [actions/setup-python](https://github.com/actions/setup-python) task.

Reviewer 2: ```suggestion
More versions will (soon!) be available to install on-the-fly through the [`setup-python`](https://github.com/actions/setup-python) action.
```

^ I changed "all" to "more" since we won't actually have all versions of Python available, right?

Reviewer 3: Thanks!

## Adding new versions

> We are trying to build and release new versions of Python as soon as they are released. However, feel free to open issue if some versions are missed.

Reviewer: Change this last sentence to just "Please open an issue if any versions are missing."

Question: should we handle issues on each of the repos or direct all issues to virtual-environments repo? I can see pros and cons of both...

Author: I would prefer to keep all issues in virtual-environments repository.
It will easier for DRI to maintain and for customers (so they won't need to find correct repo to create the issue)

> We are trying to build and release the new versions of Python as soon as they are released. However, feel free to open issue if some versions are missed.

Reviewer 2: ```suggestion
We are trying to build and release new versions of Python as soon as they are released. However, feel free to open issue if some versions are missed.
```

Reviewer 3: Thanks!
